spec/controllers/gestionnaires/dossiers_controller_spec.rb

```diff
@@ -173,6 +176,29 @@
         expect(response).to have_http_status(:ok)
       end
     end
+
+    context 'when the dossier is accepte' do
+      let(:dossier) { create(:dossier, :accepte, procedure: procedure) }
+
+      it 'it is not possible to go back to en_instruction as gestionnaire' do
+        expect(dossier.reload.state).to eq(Dossier.states.fetch(:accepte))
+        expect(response).to have_http_status(:ok)
+      end
+      context 'as administrateur' do
+        let (:current_user) { administrateur }
+        it 'it is not possible to go back to en_instruction' do
+          expect(dossier.reload.state).to eq(Dossier.states.fetch(:accepte))
+          expect(response).to have_http_status(:ok)
+        end
+      end
+      context 'as superadmin' do
+        let (:current_user) { administration }
+        it 'it is not possible to go back to en_instruction' do
```

Comment:
c’est l’inverse si j’ai bien compris :)
```suggestion
        it 'it is possible to go back to en_instruction' do
```